README.md

```diff
@@ -30,12 +30,12 @@ Alternatively, you can find the package available on
 
 Or if you use [**use-package**](https://www.emacswiki.org/emacs/UsePackage):
 
-```emacs-list
+```emacs-lisp
 (use-package editorconfig
   :ensure t
   :init
-  (add-hook 'prog-mode-hook (editorconfig-mode 1))
-  (add-hook 'text-mode-hook (editorconfig-mode 1)))
+  (add-hook 'prog-mode-hook '(editorconfig-mode 1))
```

Comment:
Whoops, I've realised this version doesn't work either... `add-hook` doesn't work like I thought it did. Possible just to use `(add-hook '____-mode-hook 'editorconfig-mode)`? As of Emacs 24.1 that will work as expected [when called non-interactively](http://emacs.stackexchange.com/a/5364).